source/guide_screego.rst

```diff
@@ -0,0 +1,148 @@
+.. highlight:: console
+
+.. author:: no-one <https://github.com/no-one>
+
+.. tag:: lang-go
+.. tag:: ports
+.. tag:: screen-sharing
+.. tag:: webRTC
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/screego.png
+      :align: center
+
+#######
+Screego
+#######
+
+.. tag_list::
+
+Screego_ is an open-source software that provides screen sharing via WebRTC. A demo can be found at https://app.screego.net.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Firewall ports <basics-ports>`
+  * :manual:`HTTPS <web-https>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+All relevant legal information can be found here:
+
+  * https://github.com/screego/server/blob/master/LICENSE
+
+Prerequisites
+=============
+
+To make the application accessible from the outside, open two `ports in the firewall <firewall_>`_:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ uberspace port add
+  Port 40130 will be open for TCP and UDP traffic in a few minutes.
+  [isabell@stardust ~]$ uberspace port add
+  Port 40131 will be open for TCP and UDP traffic in a few minutes.
+  [isabell@stardust ~]$
+
+Installation
+============
+
+Create a new directory, enter the directory you just created, download the latest version, unpack the archive, delete the archive afterwards, make the binary executable and create the file ``screego.config`` as a copy of the file ``screego.config.example``:
+
+.. note:: Replace ``1.10.5`` with the version of the `latest release`_.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir ~/screego
+ [isabell@stardust ~]$ cd ~/screego
+ [isabell@stardust screego]$ wget "https://github.com/screego/server/releases/download/v1.10.5/screego_1.10.5_linux_amd64.tar.gz"
+ [isabell@stardust screego]$ tar xvf screego_1.10.5_linux_amd64.tar.gz
+ [isabell@stardust screego]$ rm screego_1.10.5_linux_amd64.tar.gz
+ [isabell@stardust screego]$ chmod +x screego
+ [isabell@stardust screego]$ cp screego.config.example screego.config
+ [isabell@stardust screego]$
+
+Configuration
+=============
+
+Configure Screego
+-----------------
+
+.. note:: You can use the following command to generate your own ``SCREEGO_SECRET``: ``tr -dc A-Za-z0-9 < /dev/urandom | head -c 40; echo``
+
+.. warning:: Replace ``isabell`` with your username, ``X940rRflEAWtVQvtPmCKgsj1WoJ8IoNZ2tEaDSxZ`` with your secret and ``40130``/``40131`` with your ports!
```

Comment:
This looks like a real secret, people will just copy-paste it. How about `YourSecret`?

Reply:
Check out https://lab.uberspace.de/guide_gitea/ for instance, there's a secret as well.

Reply:
You're right, I've changed it. However, I used “snake case” as in the Gitea guide.